ui-v2/app/templates/components/changeable-set.hbs

```diff
@@ -1,6 +1,6 @@
 {{yield}}
 {{#if (gt items.length 0)}}
-  {{#yield-slot 'set' (block-params items)}}{{yield}}{{/yield-slot}}
+  {{#yield-slot name='set' params=(block-params items)}}{{yield}}{{/yield-slot}}
 {{else}}
-  {{#yield-slot 'empty'}}{{yield}}{{/yield-slot}}
+  {{#yield-slot name='empty'}}{{yield}}{{/yield-slot}}
 {{/if}}
```

Comment:
are newlines required in all `hbs` files? Would eslint/prettier pick them up?

Reply:
I don't think so, I don't think it matters either way tbh. I'm definitely not precious either way. There's a `ui-v2/.template-lintrc.js` that we use to disable a lot of the template linting.

A bit of history here is that template linting came along in `ember 3.something` and our templates where already using a different style to the new default template linting rules. So we just disabled certain rules for the moment at least.